docs/spec_draft.md

```diff
@@ -223,6 +224,48 @@ Produces a `result` tensor from a constant `value`.
 // %3: [(0.0, 1.0), (2.0, 3.0)]
 ```
 
+## stablehlo.cosine
+
+`stablehlo.cosine(operand) -> result`
+
+### Semantics
+
+Performs element-wise cosine operation on `operand` tensor and produces a `result` tensor.
```

Comment:
What does it mean to perform a cosine operation? What kind of precision with respect to an ideal mathematical result are we talking about? What are the corner cases? (Same questions for other ops in this PR and for sqrt/rsqrt in the neighboring PR).

Reply:
As mentioned in later comments, we will initially state, "Numeric precision is implementation-defined."